README.md

```diff
@@ -158,11 +172,16 @@ The run script will create virtual camera `/dev/video9`. You can change these se
 <!--It is supposed that there is only one web cam connected to the computer at `/dev/video0`.-->
 You can use command `v4l2-ctl --list-devices` to list all devices in your system.
 
-Run:
```

Comment:
I would leave this as is and add below a comment that one should use `--docker` option for Docker installation.